tutorials/R/Geospatial-skills/primer-raster-r/Introduction-to-Raster-Data-In-R/Introduction-to-Raster-Data-In-R.Rmd

```diff
@@ -398,8 +406,12 @@ used to quickly define a crop extent.
 
```

Comment:
I can't comment on the line above through the GitHub interface, but please add the code chunk option `comment=NA` to the above (e.g., ` ```{r cropDEM, eval=FALSE, comment=NA} `). See explanation in the comments below.